bokehjs/src/coffee/renderer/annotation/box_annotation.coffee

```diff
@@ -48,14 +82,25 @@ class BoxAnnotation extends HasParent
   default_view: BoxAnnotationView
   type: 'BoxAnnotation'
 
+  nonserializable_attribute_names: () ->
+    super().concat(['silent_update'])
+
+  update:({left, right, top, bottom}) ->
+    if @get('silent_update')
+      @attributes['left'] = left
+      @attributes['right'] = right
+      @attributes['top'] = top
+      @attributes['bottom'] = bottom
```

Comment:
It does. :) I tried that, it still triggered patch events to the server. I figured this would do for now until we work on a more ambitious solution after `0.11`